lib/megaLogger.js

```diff
@@ -80,12 +78,26 @@
     };
 
     /**
+     * Static, converts enum to text/label
+     *
+     * @private
+     * @static
+     */
+    var _intToLevel = {
+        '40': 'ERROR',
+        '30': 'WARN',
+        '20': 'INFO',
+        '10': 'LOG',
+        '0': 'DEBUG'
+    };
+
+    /**
      * Static, global log registry
      *
      * @private
      * @static
      */
-    MegaLogger._logRegistry = {};
+    var _logRegistry = {};
```

Comment:
MegaLogger._logRegistry was initially used as this, so that i can access it via the debugger/web dev tools to debug some loggers.... but now, after the development of this project is mainly into "maintenance mode", i like the change :) :+1: